templates/shortcodes/clients_showcase.html

```diff
@@ -0,0 +1,29 @@
+{% set clients = load_data(path="content/ecosystem/clients/clients.toml") %}
+<div class="featured-clients">
+    {% for name, client in clients %}
+    {% if client.featured %}
+    <div class="featured-client-card">
+        <img src="thumbnails/{{ client.thumbnail }}" alt="{{ name }} Preview">
+        <h3>{{ name }}</h3>
+        <p>{{ client.summary | safe }}</p>
+        <div class="stores-links">
+            {% if client.packages.apple_app_store.app_id and client.packages.apple_app_store.org %}
+            <a href="https://apps.apple.com/app/{{ client.packages.apple_app_store.org }}/{{ client.packages.apple_app_store.app_id }}" class="platform-pill">App Store</a>
+            {% endif %}
+            {% if client.packages.google_play_store.app_id %}
+            <a href="https://play.google.com/store/apps/details?id={{ client.packages.google_play_store.app_id }}" class="platform-pill">Google Play</a>
+            {% endif %}
+            {% if client.packages.f_droid.app_id %}
+            <a href="https://f-droid.org/packages/{{ client.packages.f_droid.app_id }}" class="platform-pill">F-Droid</a>
+            {% endif %}
+            {% if client.packages.flathub.app_id %}
+            <a href="https://flathub.org/apps/details/{{ client.packages.flathub.app_id }}" class="platform-pill">Flathub</a>
+            {% endif %}
+            {% if client.packages.other_url %}
+            <a href="{{ client.packages.other_url }}" class="platform-pill">Web</a>
+            {% endif %}
```

Comment:
This is duplicated between the card and table code, right? Maybe we should move it out into a macro.